src/components/Image/Image.tsx

```diff
@@ -0,0 +1,35 @@
+import { HTMLAttributeAnchorTarget } from "react"
+
+export interface ImageProps {
+  src?: string
```

Comment:
suggestion: this should be made required

Reply:
[fc32135](https://github.com/isomerpages/isomer-components/pull/7/commits/fc32135ffdc56f7e0147075b36f8164d36ebbef6)

similar to infopic, i think its good to make alt text required too